Bootcamp2015-AmazingRace/Views/MapPage.xaml

```diff
@@ -0,0 +1,48 @@
+﻿<Page
+    x:Class="Bootcamp2015.AmazingRace.Views.MapPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Bootcamp2015.AmazingRace.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:maps="using:Windows.UI.Xaml.Controls.Maps"
+    xmlns:converters="using:Bootcamp2015.AmazingRace.Converters"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Page.Resources>
+        <converters:BooleanToVisibilityConverter x:Key="bool2vis"/>
+    </Page.Resources>
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+
+        <maps:MapControl 
+            x:Name="_map"
+            HorizontalAlignment="Stretch" VerticalAlignment="Stretch"
+            MapServiceToken="q53_aaK2eEb2M-qo08mFpQ"
```

Comment:
Typically better to bind the token from the ViewModel, this would allow it to be configurable instead of hardcoded in the XAML.